config/config.js

```diff
@@ -3,7 +3,7 @@ const PORT = 8484;
 const SALT = 10;
 const DATABASE_NAME = "ChitChat";
 const DATA_USER_NAME = "postgres";
-const DATABASE_PASSWORD = "newPassword";
+const DATABASE_PASSWORD = "16122000";
```

Comment:
This should come from .env file, if you guys are using that. Sensitive informations like api keys, passwords, etc are written in .env files, and these .env files are not commited in git. This .env file should be registed in .gitignore similar as node_modules